PROTOCOL.md

```diff
@@ -603,6 +613,102 @@ Reader Version 2 | Respect [Column Mapping](#column-mapping)
 
 # Appendix
 
+## Deletion Vectors
+
+`add` and `remove` actions can optionally include with a Deletion Vector (DV) that provides information about logically deleted rows, that are however still physically present in the underlying data file and must thus be skipped during processing. 
+
+DVs can be stored and accessed in different ways, indicated by the `storageType` field. The Delta protocol currently supports inline or on-disk storage, where the latter can be accessed either by a relative path derived from a UUID or an absolute path.
+
+### Deletion Vector Descriptor Schema
+
+The schema of the `DeletionVectorDescriptor` struct is as follows:
+
+Field Name | Data Type | Description
+-|-|-
+storageType | String | A single character to indicate how to access the DV. (See below.)
+pathOrInlineDv | String | Three format options are currently proposed:<ul><li>If `storageType = 'u'` then  `<random prefix - optional><base85 encoded uuid>`: The deletion vector is stored in a file with a path relative to the data directory of this Delta table, and the  file name can be reconstructed from the UUID. See Derived Fields for how to reconstruct the file name. The random prefix is recovered as the extra characters before the (20 characters fixed length) uuid.</li><li>If `storageType = 'i'` then `<base85 encoded bytes>`: The deletion vector is stored inline in the log. The format used is the `RoaringBitmapArray` format also used when the DV is stored on disk and described in [Deletion Vector Format](#Deletion-Vector-Format).</li><li>If `storageType = 'p'` then `<absolute path>`: The DV is stored in a file with an absolute path given by this path, which has the same format as the `path` field in the `add`/`remove` actions.</li></ul>
```

Comment:
isnt the convention `[<random prefix>]<base85 encoded uuid>` to show that random prefix is optional?

Reply:
It could be. I didn't want anyone to think that the `[...]` are literals, so this seemed safer. (Of course, the same could be said for `<...>`...I had to pick *some* syntax :D)